### PR TITLE
#8 Add Toppings, Pizzas to the Lists

### PR DESCRIPTION
**Pull request by jordihereu**
- Adding 2 new pizzas, 5 new toppings
- Fixing compilation error (replace wrong {} by [] next to Hawaiian Elvis pizza)

### CODE REVIEW

File: Repo.cs [hunk 1]
```diff
@@ -54,7 +56,12 @@ static class Repo
                 "pineapple",
                 "spinach",
                 "tomato",
-                "saurkraut"
+                "saurkraut",
+                "riccota",
+                "parmegian",
```
Comment (javajudt) [inline Repo.cs]:
```suggestion
                "parmigian",
```

File: Repo.cs [hunk 2]
```diff
@@ -25,7 +26,8 @@ static class Repo
             {
                 { "Veggie Lovers",      new[] {"tomato", "olive", "spinach", "mushroom", "onion" } },
                 { "Spice Garden",       new[] {"green pepper", "banana pepper", "jalapeño pepper" } },
-                { "Spinach Delight",    new[] {"spinach", "tomato" } }
+                { "Spinach Delight",    new[] {"spinach", "tomato" } },
+                { "Four cheese",        new[] {"riccota","parmegian","mozzarella","gorgonzola" } }
```
Comment (javajudt) [inline Repo.cs]:
```suggestion
                { "Four cheese",        new[] {"riccota","parmigian","mozzarella","gorgonzola" } }
```

**Review by javajudt — CHANGES_REQUESTED** [review]
Just noticed a couple minor typos, otherwise good!

**Comment by jordihereu** [comment]
OK, I´ve applied the PR feedback. It´s the first time I´m solving conflicts in a GitHub branch so I did my best. I don´t know if I did it the best possible way. Anyway I think my changes are there to be reviewed.
Thanks.

**Review by javajudt — APPROVED** [review]
Looks good! Thanks!

**Pull request merged**